api/test/views/mock_keystone.py

```diff
@@ -0,0 +1,191 @@
+import json
+
+from rest_framework import status
+from rest_framework.response import Response
+from django.http import Http404
+from rest_framework.views import APIView
+from django.http import JsonResponse, HttpResponse
+
+from threepio import logger
+
+from atmosphere import settings
+from atmosphere.settings.local import AUTHENTICATION as auth_settings
+from atmosphere.settings.local import TEST as test_settings
+
+
+class MockKeystoneView(APIView):
+    """
+    This is just a simple view to catch any url we are not handling
+    """
+
+    def get(self, request):
+        logger.info(" MockKeystoneView:get called")
+        for header in request.META:
+            logger.info("   header: %s   value %s" % (header, request.META[header]))
+        return JsonResponse({'text': 'get response'})
+
+    def post(self, request):
+        logger.info("MockKeystoneView:post called")
+        logger.info(request.body)
+        return JsonResponse({'text': 'get response'})
+
+
+class MockKeystoneUsersProjects(APIView):
+    """
+    This is just a simple view to catch any url we are not handling
```

Comment:
@knikolla  Should we use port 5000 rather than 8082? Or it doesn't really matter?